Say Their Names/Source/Components/MainTabBarController.swift

```diff
@@ -9,7 +9,8 @@
 import UIKit
 
 class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
-
+    public weak var service: Service!
```

Comment:
We should remove the `!` from `Service` so we dont have to deal with optionals

Reply:
The variable is weakly reference. Removing the `!` would require it to be `?` or remove the `weak` qualifier.

Reply:
Oh thats the other thing. Do we actually need to have a weak reference to the service? The reference count should be decremented automatically when the Controller is deallocated, no?

Reply:
Indeed. As long as we keep `Service` to not contain any references to a vc... then using a strong reference is acceptable.

Reply:
Yea, I really dont think the Service should reference a VC so I think it might be a good move to make it a strong reference

Reply:
Removed.